src/drivers/imu/adis16448/adis16448.cpp

```diff
@@ -560,6 +567,16 @@ ADIS16448::ADIS16448(int bus, const char *path_accel, const char *path_gyro, con
 	_mag_scale.z_scale  = 1.0f;
 
 	memset(&_call, 0, sizeof(_call));
+
+	// allocate dma memory
+	_dma_data_buffer = (struct ADISReport *)fat_dma_alloc(13*sizeof(uint16_t));
```

Comment:
maybe a named constexpr for the 13?

Reply:
that is still horrible hacked-in - so will need to change this anyway  
This is still sort of proof-of-concept code from the very beginning - I wasn't sure if the dma_alloc etc would work at all and postponed cleaning up this to sort of now.. :-)